spike_main/spike.cc

```diff
@@ -55,6 +55,9 @@ static void help(int exit_code = 1)
   fprintf(stderr, "                        This flag can be used multiple times.\n");
   fprintf(stderr, "  --rbb-port=<port>     Listen on <port> for remote bitbang connection\n");
   fprintf(stderr, "  --dump-dts            Print device tree string and exit\n");
+  fprintf(stderr, "  --ust-trace=<file>    Write Instruction trace file\n");
```

Comment:
What does "UST" mean? Why not `--inst-trace`?

Reply:
UST is UltraSoC Trace, it's historic as the work on instruction was initially done by UltraSoC in 2017.

Reply:
So the switch is "UltraSoC Trace trace"? Why not switch this to `--inst-trace`? There's no other references to UltraSoC that I can see in this PR, so no reason to keep this historical name.

Reply:
Corrected.